parlai/core/torch_ranker_agent.py

```diff
@@ -226,6 +228,7 @@ def train_step(self, batch):
 
     def eval_step(self, batch):
         """Evaluate a single batch of examples."""
+        self.is_training = False
```

Comment:
you could do all this in batch_act of TA and then we'd have it in TGA too

Reply:
yes good idea i'll move it up a level in case we need this info elsewhere